manifests/base/deployment.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: runner-manager
+  labels:
+    app.kubernetes.io/name: runner-manager
+    app.kubernetes.io/instance: runner-manager
```

Comment:
`metadata.labels`, `spec.selector`, and `spec.template.metadata` can and should be added through base kustomization.yaml file

Reply:
ok

Reply:
adding link to doc in case it helps: https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/commonlabels/

Reply:
ok